.github/workflows/integration-build-product.yml

```diff
@@ -1190,25 +1191,48 @@ jobs:
           TF_VAR_number_agents: 0
+          # TF_VAR_proxmox_api_url: ${{ secrets.proxmox_api_url }}
+          # TF_VAR_proxmox_api_token_id: ${{ secrets.proxmox_api_token_id }}
+          # TF_VAR_proxmox_api_token_secret: ${{ secrets.proxmox_api_token_secret }}
+          # TF_VAR_provision_ssh_pem: ${{ secrets.ssh_priv_key }}
+          # TF_VAR_ip_list: '["dhcp"]'
+          # TF_VAR_storage: "local-lvm"
+          # TF_VAR_proxmox_host_list: '["mary"]'
+          # TF_VAR_vm_gateway: "10.10.1.254"
+          # TF_VAR_ip_mask: 23
+          # TF_VAR_bios: "ovmf"
+          # TF_VAR_pool: "IP-Temp-VMs"
+          # TF_VAR_tags: "ip-simul-ci"
+          # TF_VAR_fleet_hosts_user: "devops"
+          # TF_VAR_template_name: "u22dci-gpu"
+          # TF_VAR_fleet_manager_name: ${{ steps.infra_names.outputs.simul_prefix }}
+          # TF_VAR_fleet_manager_memory: 51200
+          # TF_VAR_fleet_manager_cores: 14
+          # TF_VAR_fleet_manager_disk_size: "110G"
+          # TF_VAR_fleet_manager_balloon: 0
+          TF_VAR_proxmox_ve_username: ${{ secrets.proxmox_ve_username }}
+          TF_VAR_proxmox_ve_password: ${{ secrets.proxmox_ve_password }}
+          TF_VAR_proxmox_host: "mary"
           TF_VAR_proxmox_api_url: ${{ secrets.proxmox_api_url }}
-          TF_VAR_proxmox_api_token_id: ${{ secrets.proxmox_api_token_id }}
-          TF_VAR_proxmox_api_token_secret: ${{ secrets.proxmox_api_token_secret }}
-          TF_VAR_provision_ssh_pem: ${{ secrets.ssh_priv_key }}
-          TF_VAR_ip_list: '["dhcp"]'
-          TF_VAR_storage: "local-lvm"
-          TF_VAR_proxmox_host_list: '["mary"]'
-          TF_VAR_vm_gateway: "10.10.1.254"
-          TF_VAR_ip_mask: 23
-          TF_VAR_bios: "ovmf"
-          TF_VAR_pool: "IP-Temp-VMs"
-          TF_VAR_tags: "ip-simul-ci"
-          TF_VAR_fleet_hosts_user: "devops"
-          TF_VAR_template_name: "u22dci-gpu"
-          TF_VAR_fleet_manager_name: ${{ steps.infra_names.outputs.simul_prefix }}
-          TF_VAR_fleet_manager_memory: 51200
-          TF_VAR_fleet_manager_cores: 14
-          TF_VAR_fleet_manager_disk_size: "110G"
-          TF_VAR_fleet_manager_balloon: 0
+          TF_VAR_vm_name: "test-bpg-tf"
+          TF_VAR_vm_memory: 8192
+          TF_VAR_vm_cores: 4
+          TF_VAR_vm_core_type: "host"
+          TF_VAR_vm_disk_size: 20
+          TF_VAR_vm_disk_interface: "scsi0"
+          TF_VAR_vm_type: "q35"
+          TF_VAR_vm_os_type: "l26"
+          TF_VAR_vm_network_bridge: "vmbr0"
+          TF_VAR_vm_network_ip_address: "dhcp"
+          TF_VAR_vm_disk_storage: "nas-mary"
+          TF_VAR_vm_img_id: "nas-mary:iso/0.0.1-13-desktop-jammy_local.img"
+          TF_VAR_vm_bios: "ovmf"
+          TF_VAR_cloud_init_storage: "local"
+          TF_VAR_hostpci_device: "hostpci0"
+          TF_VAR_hostpci_device_id: "0000:01:00.0"
+          TF_VAR_hostpci_device_pcie: "true"
+          TF_VAR_hostpci_device_xvga: "true"
+          # TF_VAR_hostpci_device_rombar: "true"
```

Comment:
@duartecoelhomovai  and @DenisBorzenkov I think we can find better to pass parameters that are cluster / type of VM related, maybe with predefining some variables of our cluster somewhere, for example :
```
  cluster_config:
    name: mary
    nodes:
      mary:
        ip: 10.10.1.22
        max_memory: 32G
        max_cpu: 12
        max_disk: 100G
        gpu_pci: 0000:00:00.0
        vm_storage: local-lvm
        ...
      bb8:
         ip: 10.10.1.x
        max_memory: 64G
        max_cpu: 16
        max_disk: 100G
        gpu_pci: 0000:00:00.0
        vm_storage: local-lvm
        ...
      bb9:
        ip: 10.10.1.y
        max_memory: 64G
        max_cpu: 16
        max_disk: 100G
        gpu_pci: 0000:00:00.0
        vm_storage: local-lvm
        ...     

```
please advise on where to put this config

Reply:
some idea:
```
env:
    CLUSTER_ENV_FILE: ./mary.env # or      CLUSTER_ENV_FILE: ./bb8.env    
  run: |
    source $CLUSTER_ENV_FILE

```